test/metabase/api/table_test.clj

```diff
@@ -455,8 +484,11 @@
                                                               :updated_at   $
                                                               :id           $
                                                               :raw_table_id $
-                                                              :created_at   $}))}))}])
-  ((user->client :rasta) :get 200 (format "table/%d/fks" (id :users))))
+                                                              :created_at   $}))}))}]))
+(do (fingerprint/cache-field-values-for-table! (Table (id :users)))
```

Comment:
other bad ideas:

treating code indentation as optional and just starting from column 1 when you feel like it